do

```diff
@@ -98,8 +142,14 @@ def create_parser():
     )
     ibp.set_defaults(func=import_batch)
 
+    # TODO: candidate to be split into two steps: "integrate data" -> export
     exp = sp.add_parser("web_export", help="Create data export for web.")
     exp.add_argument("-c", "--comment", help="A short comment (to be part of path).")
+    exp.add_argument("models_file", help="A result HDF file of import_gleam_batch step")
+    exp.add_argument("rates", help="Path to a file for hospital rates (M49 indexed)")
+    exp.add_argument("john_hopkins", help="John Hopkins data (UN code indexed)")
+    exp.add_argument("foretold", help="Foretold data (UN code indexed)")
+    # TODO: additional data files we want to have in the export
```

Comment:
I would prefer to add all of those paths to `config.yaml` - they are not changing over time (and can still be overwritten there). Also the usage gets much easier.

Reply:
Yeah, that's what I meant by this in the PR description:

> not loading each file independently

it's mostly for me to be explicit about what is being used where. I am going to put it in config to follow the codebase standard when done.

But. Doing it via config makes understanding the codebase and functionality much harder. I spent 30 minutes finding out what's the file you share on slack. If it was an argument, it would be much easier. Config is just a global state used in this module loading files on various places (e.g. https://github.com/epidemics/epimodel/pull/25/files#diff-d4579b2688d675235f402f6b4b43bcbfR182) or loading config here: https://github.com/epidemics/epimodel/pull/25/files#diff-d4579b2688d675235f402f6b4b43bcbfR181 . The usage is easier only _if_ you have all the files available, which is not the case until you clone the repo, run gleamviz and maybe some other magic I am not aware of because it's hardcoded in the codebase. It's unclear what's needed in the given step then.

Reply:
Ok, here is my mental model:
* Gleam export (the `batch-something.hdf5`) is a good candidate for an argument, it may be passed around in the data github repo or otherwise (it is not small, and newer versions do not really subsume the old ones).
* regions, timezones, foretold, john-hopkins, and (I assume) rates have a stable space in the data repo. One may want to e.g. change the data directory (since you have the data git repo elsewhere, but that fits the config quite well, as this is long-term).

Feel free to add override options, but I have done enough composing and copy-pasting 2-line command-lines in epifor to steer clear of having to specify something that has good defaults.

Reply:
Why boycott global state so much, Dan?

Esp. when it is clear where it comes from:
* args comes from argparse in the same file
* config comes from config.yaml, loaded just below (I agree that it would be better passed arond explicitly rather than in args, similarly with rds - would that help?)

Reply:
> Ok, here is my mental model: [...]

Ok, will follow it, that sounds good to me :+1: .

> Esp. when it is clear where it comes from:

This condition was not true when I started to work on this task and it's _the_ problem of global state :grin: . It's all clear to you, but I claim that it's not clear to not anyone who starts on this. I am happy to bet that 90% of new coders will fail on missing `data/regions.csv` (the same way I did) which gets silently loaded in main... Or the whole data directory actually.

Reply:
Done: https://github.com/epidemics/epimodel/pull/25/commits/8fef0dd1b9fc52b38f0ca37aafc61c7f7f6e8842